remote-api/src/main/java/org/alfresco/rest/api/impl/AuditImpl.java

```diff
@@ -913,7 +913,7 @@ public int getAuditEntriesCountByAppAndProperties(AuditService.AuditApplication
         final String applicationName = auditApplication.getKey().substring(1);
 
         AuditQueryParameters parameters = new AuditQueryParameters();
-        parameters.setApplicationName(applicationName);
+        parameters.setApplicationName(auditApplication.getName());
```

Comment:
Following the previous suggestion, we could supply the `applicationName` variable again here.

Reply:
Changes are done as per the comment.